week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -1,45 +1,47 @@
 // The below functions are syntactically correct but not outputting the right results.
 // Look at the tests and see how you can fix them.
 
+
 function mood() {
   let isHappy = true;
-
-  if (isHappy) {
+  if (!isHappy) {
     return "I am happy";
   } else {
     return "I am not happy";
   }
 }
 
+
 function greaterThan10() {
   let num = 10;
-  let isBigEnough;
-
-  if (isBigEnough) {
+  let isBigEnough = num;
+  if (isBigEnough >= num) {
     return "num is greater than or equal to 10";
   } else {
     return "num is not big enough";
   }
 }
 
+
 function sortArray() {
   let letters = ["a", "n", "c", "e", "z", "f"];
   let sortedLetters;
-
+  sortedLetters = letters.sort();
   return sortedLetters;
 }
 
+
 function first5() {
   let numbers = [1, 2, 3, 4, 5, 6, 7, 8];
   let sliced;
-
+  sliced = numbers.slice(0, 5);
```

Comment:
we need the first 5, we started slicing from 0.
`0 , 1 , 2 , 3, 4 `
how many numbers are there ??